catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -555,8 +557,12 @@ public Table createTable(
       Map<String, String> properties,
       Transform[] partitioning,
       Distribution distribution,
-      SortOrder[] sortOrders)
+      SortOrder[] sortOrders,
+      Index[] indexes)
       throws NoSuchSchemaException, TableAlreadyExistsException {
+    Preconditions.checkArgument(
+        indexes.length == 0,
+        "Hive-catalog does not support indexes, current Gravitino hive-catalog only supports Hive 2.x");
```

Comment:
The error message should be modified, since [indexing was removed since 3.0](https://cwiki.apache.org/confluence/display/Hive/IndexDev)